environments/stg/stg.tfvars

```diff
@@ -4,7 +4,7 @@ env                    = "aat"
 subscription           = "stg"
 certificate_name_check = false
 
-app_gw_private_ip_address = ["10.10.24.121", "10.10.24.125"]
+app_gw_private_ip_address = ["10.10.24.125"]
```

Comment:
This is wrong value, in AAT we have two app gateways.

You need to remove from `cft_apps_cluster_ips`

Reply:
now corrected.

Reply:
Cheers @darren221980